integration-tests/multer.spec.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const { assert } = require('chai')
+const path = require('path')
+const axios = require('axios')
+
+const {
+  createSandbox,
+  FakeAgent,
+  spawnProc
+} = require('./helpers')
+
+describe('Suspicious request blocking - multer', () => {
+  let sandbox, cwd, startupTestFile, agent, proc, env
+
+  ['1.4.4-lts.1', '1.4.5-lts.1'].forEach((version) => {
+    describe(`v${version}`, () => {
+      before(async () => {
+        sandbox = await createSandbox(['express', `multer@${version}`])
+        cwd = sandbox.folder
+        startupTestFile = path.join(cwd, 'multer/index.js')
```

Comment:
as we are using `path.join`, I think that using multiple args instead of a `/` is safer for windows (maybe these tests are never executed in windows...)
```suggestion
        startupTestFile = path.join(cwd, 'multer', index.js')
```